debug/index.js

```diff
@@ -18,19 +18,29 @@ insertCss(
 
 var MapboxGeocoder = require('../');
 
-var mapDiv = document.body.appendChild(document.createElement('div'));
-mapDiv.style.position = 'absolute';
-mapDiv.style.top = 0;
-mapDiv.style.right = 0;
-mapDiv.style.left = 0;
-mapDiv.style.bottom = 0;
-
-var map = new mapboxgl.Map({
-  container: mapDiv,
-  style: 'mapbox://styles/mapbox/streets-v9',
-  center: [-79.4512, 43.6568],
-  zoom: 13
-});
+// var mapDiv = document.body.appendChild(document.createElement('div'));
```

Comment:
What about creating a different debug page such as `debug/mapless.js` so both can be tested side by side?

Reply:
Yeah great idea, I'll add that to this PR.